kore/src/main/scala/org/kframework/definition/outer.scala

```diff
@@ -348,10 +348,6 @@ case class Module(val name: String, val imports: Set[Module], localSentences: Se
 
   override lazy val hashCode: Int = name.hashCode
 
-  override def equals(that: Any) = that match {
```

Comment:
If we're going to be using the default equals for case classes, let's use the default hashCode as well.

Reply:
I thought about that. But I think this is faster and still safe.
It's only used to quickly determine if two modules are different. And in our case, that happens most of the time since a Definition has unique Module names.